### PR TITLE
Add `source` entry for ros2_openai_server (jazzy)

### PR DESCRIPTION
**Pull request by AndyZe**
This is a new package so I'm adding the `source` entry per https://docs.ros.org/en/jazzy/How-To-Guides/Releasing/Release-Team-Repository.html#create-a-new-release-repository

@brettpac @yassiezar

### CODE REVIEW

**Comment by AndyZe** [comment]
I'd be fine with `chatgpt_server` if that floats your boat. I'm aware that I should drop `ros2_` from the front

File: jazzy/distribution.yaml [hunk 1]
```diff
@@ -6043,6 +6043,12 @@ repositories:
       url: https://github.com/Kinovarobotics/ros2_kortex.git
       version: main
     status: developed
+  ros2_openai_server:
+    source:
```
Comment (Yadunund) [inline jazzy/distribution.yaml:6047]: Do you also want to add a doc entry?

**Review by Yadunund — CHANGES_REQUESTED** [review]
- [x] At least one of the following must be present
  - [x] Top level license file:
  - [ ] Per package license files:
- [x] License is [OSI-approved](https://opensource.org/licenses):  Apache License 2.0
- [x] License correctly listed in package.xmls
- [x] Public source repo: https://github.com/robosoft-ai/ros2_openai_server/tree/jazzy
- [x] Source repository contains ROS packages
- [ ] Each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions

<details><summary>Package name details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<name>" "{}" ";"
<name>ros2_openai_server</name>
```
</details>

<details><summary>License details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<license>" "{}" "+"
<license>Apache-2.0</license>
```
</details>

Echoing @marcoag sentiments I too think the naming is very generic and does not pass [REP 144](https://www.ros.org/reps/rep-0144.html). It would be good to properly suffix the package, eg. `robosoft_ai_openai_server`. (`ros2` can be dropped imo but not a strict requirement).

File: jazzy/distribution.yaml (hunk 1 above)
Comment (AndyZe) [reply jazzy/distribution.yaml:6047]: sure

**Comment by AndyZe** [comment]
OK, I can live with `robosoft_openai_server` or `robosoft_openai`

**Review by marcoag — APPROVED** [review]

**Review by Yadunund — APPROVED** [review]
LGTM, thanks for iterating!

**Pull request merged**